widget/check_group.go

```diff
@@ -44,6 +46,19 @@ func (r *CheckGroup) Append(option string) {
 	r.Refresh()
 }
 
+// Remove removes the first specified option found from a CheckGroup widget.
+//
+// Since: 2.3
+func (r *CheckGroup) Remove(s string) bool {
```

Comment:
I would prefer to call the parameter `option` yo be consistent with `.Append`.

Reply:
Done. Good catch.